s30904-JakubTreder/zajecia3/zadanie4_zajecia3.txt

```diff
@@ -0,0 +1,9 @@
+1. 
```

Comment:
Pięknie. Wykonane polecenia. Super... tylko czego się tyczą? Gdyby było wymienione zadanie było by to oczywiste.